graders/python/requirements.txt

```diff
@@ -1,25 +1,25 @@
-automata-lib==7.1.0
+automata-lib==8.0.0
```

Comment:
Discussed with @nwalters512, we think we can actually remove `automata-lib` from the grader image.

Reply:
Restored to 7.1.0. I'll leave the discussion on removing it to a different PR, possibly #8223.